data/mods/Magiclysm/Spells/magus.json

```
@@ -1,4 +1,28 @@
 [
+  {
+    "id": "shadow_field",
+    "type": "SPELL",
+    "name": "Shadow Field",
+    "description": "Cast a field of shadows.",
```

```suggestion
    "description": "Cast a field of intense shadows.",
```
To take into account the change to `fd_darkness` (if not simply changing all mentions of "shadow" to "darkness").